dsync_test.go

```diff
@@ -92,8 +92,8 @@ func TestMain(m *testing.M) {
 	}
 
 	nclnts = make([]NetLocker, 15)
-	if _, err := New(nclnts, 0); err == nil {
-		log.Fatalf("Should have failed")
+	if _, err := New(nclnts, 0); err != nil {
+		log.Fatalf("Should pass")
```

Comment:
you should add more tests to check for odd number of nodes.. validate the dquorum and dquorumReads values as well.